frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -146,13 +146,14 @@ Feature: Pipeline Runs
              Then user is able to see expanded logs page
 
 
-        @regression
+        @regression @to-do
         Scenario: kebab menu options in pipeline Runs page: P-07-TC14
             Given pipeline run is displayed for "pipeline-aaa" without resource
              When user selects the Pipeline Run for "pipeline-aaa"
               And user navigates to pipelineRuns page
+              And user selects "Start" option from kebab menu for pipeline "pipeline-aaa"
               And user selects the kebab menu in pipeline Runs page for pipeline "pipeline-aaa"
-             Then user is able to see kebab menu options Rerun, Delete Pipeline Run
+             Then user is able to see kebab menu options Stop, Cancel, Rerun, Delete Pipeline Run
```

Comment:
`Stop` and `Cancel` options will be available until the PipelineRun is running. If PipelineRun is successful then only `Rerun`, `Delete` options will be available.

Reply:
This is for the running pipeline

Reply:
Will this also be affected by pipeline version 1.8?

Reply:
in Pipeline operator version 1.8. `Stop` and `Cancel` options will be available until the PipelineRun is running.